rclcpp/complex_msg_publisher/main.cpp

```diff
@@ -27,8 +27,9 @@ class complex_msg_publisher_cpp_node : public rclcpp::Node
   complex_msg_publisher_cpp_node()
   : Node("complex_msg_publisher_cpp")
   {
+    rclcpp::QoS qos(10);
     publisher_ = this->create_publisher<complex_msgs::msg::NestedMsgTest>(
-      "complex_msgs_msg_NestedMsgTest");
+      "complex_msgs_msg_NestedMsgTest", qos);
```

Comment:
You may want to use rclcpp::SystemDefaultsQoS() instead.

Reply:
Change!